### DISCUSSION TITLE
Works perfectly with Gopass and Git Sync

### DISCUSSION THREAD

**Discussion by vhscom** [zeapo/Android-Password-Store#1744] [Show and tell]
I recently moved away from KeePass + Bitwarden (for mobile) to Password Store and gopass. I just want to tell you how good it feels to eliminate the need for a master password (honeypot/single-point-of-failure) from my cross-device password sync workflow. Great job on this app!! https://vhs.codeberg.page/post/gopass-git-password-management/

**Comment by msfjarvis** [comment]
Thank you! Always lovely to hear from users :)

Please file issues if you find anything lacking.
